docs/commands/ft.aggregate.md

```diff
@@ -1,120 +1,176 @@
-#### Complexity
+---
+syntax: 
+---
 
-Non-deterministic. Depends on the query and aggregations performed, but it is usually linear to the number of results returned.
+Run a search query on an index, and perform aggregate transformations on the results, extracting statistics etc from them
 
----
+## Syntax
+
+{{< highlight bash >}}
+FT.AGGREGATE index query [VERBATIM] [ LOAD count field [field ...]] [TIMEOUT timeout] [LOAD *] [ GROUPBY nargs property [property ...] [ REDUCE function nargs arg [arg ...] [AS name] [ REDUCE function nargs arg [arg ...] [AS name] ...]] [ GROUPBY nargs property [property ...] [ REDUCE function nargs arg [arg ...] [AS name] [ REDUCE function nargs arg [arg ...] [AS name] ...]] ...]] [ SORTBY nargs [ property ASC | DESC [ property ASC | DESC ...]] [MAX num]] [ APPLY expression AS name [ APPLY expression AS name ...]] [ LIMIT offset num] [FILTER filter] [ WITHCURSOR [COUNT read_size] [MAXIDLE idle_time]] [ PARAMS nargs name value [ name value ...]] [DIALECT dialect]
```

Comment:
We have to remove syntax from headings because it takes up a lot of space and interferes with reading. We have a number of issues filed against this. The syntax is the same as what's generated from json. I just copied it.